docs/server-options.md

```diff
@@ -0,0 +1,111 @@
+---
+id: server-options
+title: Temporal server options
+sidebar_label: Start options
+---
+
+## Overview
+
+To run the Temporal, include the temporal server package `go.temporal.io/server/temporal` in your Go application. Then use it to create and start a new server:
+
+```go
+s := temporal.NewServer()
+err := s.Start()
+if err != nil{
+	log.Fatal(err)
+}
+```
+
+## Options
+
+`NewServer()` accepts functions as parameters. Each function returns a `ServerOption` that is applied to the instance.
+
+### WithConfig
+
+Overrides the default configuration with a custom configuration that is defined in the config package `go.temporal.io/server/common/service/config`.
+
+```go
+s := tempora.NewServer(
+	temporal.WithConfig(cfg),
+)
+```
+
+### WithConfigLoader
+
+Load a custom configuration from a file.
+
+```go
+s := tempora.NewServer(
+	temporal.WithConfigLoader(configDir, env, zone),
```

Comment:
maybe spell out where `configDir` is

Reply:
Eventually, these will be converted to real samples via snipsync which will include a link to the source which will create a traceable path to these parameters.
For now, I have added a link to the server source code with the functions.